practice/my_utils/exception_practice.py

```diff
@@ -0,0 +1,20 @@
+def exception_1(arg1: int) -> int:
+    try:
+        is_positive_number = func(arg1)
+        if is_positive_number == False:
```

Comment:
```suggestion
        if not is_positive_number:
```
pythonic하다는 건 이렇다고 하네요 ㅎ